packages/core/admin/admin/src/pages/AuthPage/utils/forms.js

```diff
@@ -58,21 +58,19 @@ const forms = {
     fieldsToDisable: ['email'],
     fieldsToOmit: ['userInfo.confirmPassword', 'userInfo.news', 'userInfo.email'],
     schema: yup.object().shape({
-      userInfo: yup.object().shape({
-        firstname: yup.string().required(translatedErrors.required),
-        lastname: yup.string().required(translatedErrors.required),
-        password: yup
-          .string()
-          .min(8, translatedErrors.minLength)
-          .matches(/[a-z]/, 'components.Input.error.contain.lowercase')
-          .matches(/[A-Z]/, 'components.Input.error.contain.uppercase')
-          .matches(/\d/, 'components.Input.error.contain.number')
-          .required(translatedErrors.required),
-        confirmPassword: yup
-          .string()
-          .oneOf([yup.ref('password'), null], 'components.Input.error.password.noMatch')
-          .required(translatedErrors.required),
-      }),
+      firstname: yup.string().required(translatedErrors.required),
```

Comment:
The change is only one level of nesting less right?

Reply:
Exactly 😅